dev-docs/bidders/zeta_global_ssp.md

```diff
@@ -29,9 +29,8 @@ The Zeta Global SSP adapter requires setup and approval from the Zeta Global SSP
 {: .table .table-bordered .table-striped }
 | Name                 | Scope    | Description                                                                                                         | Example      | Type      |
 |----------------------|----------|---------------------------------------------------------------------------------------------------------------------|--------------|-----------|
-| `sid`                | required | Seller ID. The identifier associated with the seller or reseller account within the advertising system              | `"sid123"`   | `string`  |
-| `shortname`          | required | Publisher Name. The unique name associated with the seller or reseller account within the advertising system        | `"pub_name"` | `string`  |
-| `tagid`              | required | Identifier for specific ad placement or ad tag that was used to initiate the auction                                | `"123877"`   | `string`  |
+| `sid`                | required | Seller ID. The identifier associated with the seller or reseller account within the advertising system              | `"123"`      | `integer` |
```

Comment:
example demonstrates string instead of integer

Reply:
Fixed, thank you